ice/recipes/meta/eval_paper_qa/qa.py

```diff
@@ -0,0 +1,29 @@
+from fvalues import F
+
+from ice.recipe import recipe
+
+
+DEFAULT_CONTEXT = "We're running a hackathon on 9/9/2022 to decompose complex reasoning tasks into subtasks that are easier to automate & evaluate with language models. Our team is currently breaking down reasoning about the quality of evidence in randomized controlled trials into smaller tasks e.g. placebo, intervention adherence rate, blinding procedure, etc."
+
+DEFAULT_QUESTION = "What is happening on 9/9/2022?"
+
+
+def make_qa_prompt(context: str, question: str) -> str:
+    return F(
+        f"""
+Background text: "{context}"
+
+Answer the following question about the background text above:
+
+Question: "{question}"
+Answer: "
+"""
+    ).strip()
+
+
+async def answer(
+    context: str = DEFAULT_CONTEXT, question: str = DEFAULT_QUESTION
+) -> str:
+    prompt = make_qa_prompt(context, question)
+    answer = await recipe.agent().complete(prompt=prompt, stop='"')
+    return answer
```

Comment:
Rename to `qa_baseline.py` so it's clear this is a baseline, not some other part of the meta recipe